### PR TITLE
Simplify package name.

### PR DESCRIPTION
**Pull request by hunterhector**
This PR fixes https://github.com/asyml/forte-wrappers/issues/43. 

### Description of changes
1. Rename the package name to `forte` and fix the corresponding README.
2. Update some name conventions (use `ftx` for ontology name, it stands for `forte ontology extension`.
3. Remove version file and init files in the root, and use `find_packages` instead.
4. Update file format with `black`: https://pypi.org/project/black/

### Possible influences of this PR.
1. Other code that imports with `forte_wrapper` will no longer work.

### Test Conducted
Describe what test cases are included for the PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,34 +30,34 @@ pip install ."[allennlp]"
...
+- [Elastic Search](https://www.elastic.co/) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte/elastic))
     - Elastic Indexer
     - Elastic Search
-- [Faiss](https://github.com/facebookresearch/faiss) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte_wrapper/faiss))
+- [Faiss](https://github.com/facebookresearch/faiss) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte/faiss))
     - Faiss Indexer
-- [GPT2](https://openai.com/blog/gpt-2-1-5b-release/) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte_wrapper/gpt2))
+- [GPT2](https://openai.com/blog/gpt-2-1-5b-release/) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte/gpt2))
     - GPT2 Text Generation
 - [Tweepy](https://docs.tweepy.org/en/latest/index.html) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte_wrapper/twitter))
```
Comment (jennyzhang-petuum) [inline README.md:62]:
```suggestion
- [Tweepy](https://docs.tweepy.org/en/latest/index.html) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte/twitter))
```

File: docs/get_started.md [hunk 2]
```diff
@@ -30,38 +30,38 @@ pip install ."[allennlp]"
...
+- [Elastic Search](https://www.elastic.co/) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte/elastic))
     - Elastic Indexer
     - Elastic Search
-- [Faiss](https://github.com/facebookresearch/faiss) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte_wrapper/faiss))
+- [Faiss](https://github.com/facebookresearch/faiss) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte/faiss))
     - Faiss Indexer
-- [GPT2](https://openai.com/blog/gpt-2-1-5b-release/) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte_wrapper/gpt2))
+- [GPT2](https://openai.com/blog/gpt-2-1-5b-release/) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte/gpt2))
     - GPT2 Text Generation
 - [Tweepy](https://docs.tweepy.org/en/latest/index.html) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte_wrapper/twitter))
```
Comment (jennyzhang-petuum) [inline docs/get_started.md:60]:
```suggestion
- [Tweepy](https://docs.tweepy.org/en/latest/index.html) ([Processors](https://github.com/asyml/forte-wrappers/tree/main/forte/twitter))
```

**Review by jennyzhang-petuum — COMMENTED** [review]
Perhaps twitter related processors haven't been changed folder name to forte?

File: README.md (hunk 1 above)
Comment (hunterhector) [reply README.md]: updated, thanks!

**Comment by hunterhector** [comment]
> Perhaps twitter related processors haven't been changed folder name to forte?

Right, I didn't merge with master when doing the renaming. Now it should be done.

**Review by jennyzhang-petuum — APPROVED** [review]

**Pull request merged**